aip/general/0001.md

```diff
@@ -28,6 +28,25 @@ serve as the source of truth for API-related documentation at Google and the
 means by which API teams discuss and come to consensus on API guidance. AIPs
 are maintained as Markdown files in the [AIP GitHub repository][].
 
+## Scope
+
+It doesn't make sense for *every* API to follow the AIPs - not even all
+*Google* APIs. Some relevant factors to consider include:
+
+- Who is writing code to call the APIs? Some AIPs (e.g. [141](AIP-141)) include
+  requirements with an expectation of APIs being called from client libraries
+  written in diverse programming languages. If the API producer is confident
+  that the API only needs to be called from Google-authored code, some of those
```

Comment:
Is this referring to non-public APIs?

Reply:
We have some APIs which are public, but not actually intended to be called by non-Google code. (In those cases the protos are published but we typically don't generate client libraries.)

Reply:
Ah, such as undocumented APIs purely for our web and mobile clients?

Reply:
Those would count as internal (I think) and not published. There are some that are expected to be called by agent processes on customers' GKE projects, as a slightly different example. I wish I had all the examples I've seen to hand...